open-ce/build_tree.py

```diff
@@ -93,6 +93,9 @@ def _create_recipes(repository, recipes, variant_config_files, variants, channel
     os.chdir(repository)
 
     config_data, _ = build_feedstock.load_package_config()
+    feedstock_conda_build_config_file = build_feedstock.get_conda_build_config()
+    if feedstock_conda_build_config_file:
+        variant_config_files.append(feedstock_conda_build_config_file)
```

Comment:
Was the recipe config file not being used?
We probably don't want to modify `variant_config_files`, because this will change the argument where this function gets called.

Reply:
Yes, earlier it wasn't being used due to which the contents in those files weren't used while creating build_tree. In my case,versions for select packages of xgboost, tf or pytorch weren't being populated. So, this fixes that problem.